templates/navigation/breadcrumb.html.twig

```diff
@@ -10,8 +10,21 @@
  * @see ./core/themes/stable/templates/navigation/breadcrumb.html.twig
  */
 #}
+
 {% if segments %}
-  {% include '@ecl/ec-component-breadcrumb' with {
-    'segments': segments,
+  {% set _breadcrumb_links = {} %}
+  {% for breadcrumb_item in segments %}
+    {% set _links = {
+      'label' : breadcrumb_item.label,
+      'path' : breadcrumb_item.href
+    } %}
+    {% set _breadcrumb_links = _breadcrumb_links|merge([_links]) %}
+  {% endfor %}
+
+  {% include '@ecl/breadcrumb' with {
+    'links' : _breadcrumb_links,
+    'navigation_text' : '',
+    'ellipsis_label': '',
+    'icon_file_path' : '/dist/images/icons/sprites/icons.svg'
```

Comment:
Shouldn't we use the variables you set in the preprocess function?

Reply:
I don't think, because this value is static and very specific of the theme.

Reply:
Ok I will change that.